src/main/java/com/imaee/propinq/buildings/services/implementations/BuildingService.java

```diff
@@ -0,0 +1,38 @@
+package com.imaee.propinq.buildings.services.implementations;
+
+import com.imaee.propinq.buildings.controllers.responses.BuildingResponse;
+import com.imaee.propinq.buildings.data.models.Building;
+import com.imaee.propinq.buildings.data.repositories.IBuildingRepository;
+import com.imaee.propinq.buildings.mappers.BuildingMapper;
+import com.imaee.propinq.buildings.services.interfaces.IBuildingService;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public class BuildingService implements IBuildingService {
```

Comment:
Estoy casi seguro de que esto no debería compilarte porque no tiene la anotación del Bean. En cualquier caso, dejemos explícito que esto es un _service_

```suggestion
@Service
public class BuildingService implements IBuildingService {
```